### PR TITLE
new components created; added save item feature

### PR DESCRIPTION
**Pull request by a5h101**
[making server accesible from all any host]
[added save item feature]
[improved login logic]
[new component for item card]
[new component for user's details]

### CODE REVIEW

File: Rentit-backend/config.env [hunk 1]
```diff
@@ -3,4 +3,5 @@ PORT=2000
 DATABASE =mongodb+srv://a5h101:<PASSWORD>@cluster0.cyfx5wk.mongodb.net/Rentit?retryWrites=true&w=majority
```
Comment (PatelNeelMahesh) [inline Rentit-backend/config.env:3]: Add `config.env` to `.gitignore` and delete from github. It contains sensitive details you should not make it available on GitHub.

File: Rentit-frontend/src/pages/itemDetail/itemDetai.jsx [hunk 2]
```diff
@@ -9,8 +9,31 @@ const ItemDetail = () => {
   const [product, setProduct] = useState(null);
   const [save, setSave] = useState(false);
   const url = `http://127.0.0.1:2000/api-rentit/v1/items/${id}`;
+  const urlSave = `http://127.0.0.1:2000/api-rentit/v1/itemDetail`;
```
Comment (PatelNeelMahesh) [inline Rentit-frontend/src/pages/itemDetail/itemDetai.jsx:12]: Add `http://127.0.0.1:2000` to `.env`.

File: Rentit-frontend/src/pages/login/login.jsx [hunk 3]
```diff
@@ -16,13 +17,6 @@ const Login = () => {
   const [loginData, setLoginData] = useState(null);
   const url = 'http://127.0.0.1:2000/api-rentit/v1/login';
```
Comment (PatelNeelMahesh) [inline Rentit-frontend/src/pages/login/login.jsx:18]: Add `http://127.0.0.1:2000` to `.env` when you deploy, change the domain you won't have to make changes in multiple files.

**Review by PatelNeelMahesh — COMMENTED** [review]
You can merge pull request after making the required changes.

**Comment by a5h101** [comment]
I have added all urls in .env file in next commit

**Pull request merged**